frame/vamm/src/tests.rs

```diff
@@ -0,0 +1,28 @@
+#[allow(unused_imports)]
+
+use crate::mock::{
+    ALICE, Event, ExtBuilder, Vamm, Origin, System
+};
+use crate::pallet;
+
+use frame_support::assert_ok;
+
+#[test]
+fn call_test_extrinsic() {
```

Comment:
I'm probably stating the obvious, but this won't work cause there are no extrinsics. Same for `test_extrinsic_emits_event()`

Reply:
Thanks!